docs/design/coreclr/jit/ryujit-tutorial.md

```diff
@@ -68,6 +67,10 @@ The JIT and the VM each implement an interface that abstracts the dependencies t
 This is the 10,000 foot view of RyuJIT. It takes in MSIL (aka CIL) in the form of byte codes, and the Importer phase transforms these to the intermediate representation used in the JIT. The IR operations are called "GenTrees", as in "trees for code generation". This format is preserved across the bulk of the JIT, with some changes in form and invariants along the way. Eventually, the code generator produces a low-level intermediate called InstrDescs, which simply capture the instruction encodings while the final mappings are done to produce the actual native code and associated tables.
 
 ### RyuJIT Phases
+
+Note: Various details in what follows are outdated and should be mostly taken to be of historical interest.
+For more up-to-date information about the phases and their details, please see [the overview document](ryujit-overview.md).
```

Comment:
There are lots of wrong details in the below, but it looked like a laborious task to get it all up-to-date, so for now I've just left this breadcrumb.